charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -78,6 +78,8 @@ spec:
         - name: credentials
           secret:
             secretName: dapr-trust-bundle
+      nodeSelector:
+        kubernetes.io/os: linux
```

Comment:
Can we define `linux` as values instead of hardcoding it?

Reply:
+1. This has to be a value because this assumes the Dapr control plane pods can't run on Windows nodes even if there's a Windows container image.

If the Dapr Control plane runs on Linux and Windows nodes, the user would need to override two values: The `nodeselector` to have the pods be deployed to `kubernetes.io/os: windows` labeled nodes, and the `image` tag to have the images be the Windows images.

Reply:
Changes added